### PR TITLE
Update React Native to v0.42.0

### PR DESCRIPTION
**Pull request by epitaphmike**

### CODE REVIEW

**Comment by Kapeli** [comment]
Please redo your fork.

**Comment by epitaphmike** [comment]
What's wrong with the fork?

**Comment by Kapeli** [comment]
No idea (I'm not that good with git), but your commit contains DraftJS.tgz and some other docsets unrelated to React Native.

**Comment by Kapeli** [comment]
Argh, sorry. I just read the title and assumed you're only changing React Native and the other commits were not intentional.

**Comment by epitaphmike** [comment]
There are 3 commits, to 3 different docsets.

File: docsets/DraftJS/docset.json [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+        "Editor"
+      ],
+    "specific_versions": [
+        {
+          "version": "0.10.0",
+          "archive": "versions/0.10.0/DraftJS.tgz"
+        },
+        {
+          "version": "0.1.0",
+          "archive": "versions/0.10.0/DraftJS.tgz"
```
Comment (Kapeli) [inline docsets/DraftJS/docset.json]: What's up with this version? Is 0.1.0 and 0.10.0 the same for DraftJS?

**Comment by epitaphmike** [comment]
Nope. The original version was 0.1.0, they've made a number of changes over
the last year and are on v0.TEN.0

**Comment by epitaphmike** [comment]
Wait... there is a typo there. Oops.

**Comment by Kapeli** [comment]
Redux docset: if you change the archive without changing the version, then users don't get updated to the latest docset. If you want to trigger an update for this docset, add something in the version number after a `/`. For example: "3.6.0/blabla". Anything that comes after the `/` isn't displayed to the user anywhere.

**Comment by epitaphmike** [comment]
Will do. I'll update both shortly

**Pull request closed without merging**